.env.sample

```diff
@@ -1,3 +1,6 @@
 TESTNET_ACCOUNT_PRIVATE_KEY=private key for an account we have on testnet that has eth.
 TESTNET_CHAIN_URL=url to connect to testnet
 LOCAL_NETWORK_ACCOUNT_PRIVATE_KEY=private key for an account we have on our local network that has eth.
+VALIDATOR1=address for validator running on node called liberty-chain
```

Comment:
Do we want to have these in the same file as the env for other purposes? I wonder if these should just be in the docker directory (and just make sure the copy command doesn't include it through ignore or individual file inclusion)